public/stylesheets/app/editor.less

```diff
@@ -504,3 +504,8 @@
 	height: auto;
 	border-bottom: 1px solid @modal-header-border-color;
 }
+
+// Widen autocomplete popup
+.ace_autocomplete {
+	width: 380px !important;
```

Comment:
I see the problem that this is trying to solve, but I don't think this is the right way to do it. Even with this, it doesn't take much of a file name before it doesn't fit, and this makes it oddly wide for the rest of the auto-complete behaviour. Is there any possibility of dynamically resizing the pop-up to the content size?